Assets/MixedRealityToolkit.Tests/EditModeTests/Core/TestFixture_01_MixedRealityToolkitTests.cs

```diff
@@ -0,0 +1,497 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This looks like  a new file that might not need to be added

Reply:
Oh weird! Good catch!